src/main/java/com/spotinst/sdkjava/model/bl/gcp/OceanGKEStrategy.java

```diff
@@ -0,0 +1,55 @@
+package com.spotinst.sdkjava.model.bl.gcp;
```

Comment:
Class and variable naming convention and file location need to be changed.

Reply:
Resolved